GettingStarted.md

```diff
@@ -0,0 +1,178 @@
+
+For more advanced and complete documentation, you might prefer the [DOCUMENTATION](DOCUMENTATION.md).
+
+# Getting started with impress.js
+## Installation / acquiring the framework
+First of all, you need to know, if you are going to have WiFi connection when you hold your presentation. If you are not sure, please use the method where you download the file instead of the cdn.
+
+
+### Including from cdn
+Loading the script from the cdn is quite straight forward. You need to include a script tag in the head of the HTML document or where ever you want to include it.:
+
+```
+<script src="https://cdn.jsdelivr.net/gh/impress/impress.js@2.0.0/js/impress.js"></script>
+```
+
+Note: in order for this to work, you need to be sure that you have a WiFi connection when giving your presentation! If you are sure, then you may use this variant. But be sure to remove 
+
+```
+<script type="text/javascript" src='./impress.js></script>
+```
+from the below example.
+
+**Direct links to different versions of the impress.js file**
+- V2.0.0: https://cdn.jsdelivr.net/gh/impress/impress.js@2.0.0/js/impress.js
+- V1.1.0: https://cdn.jsdelivr.net/gh/impress/impress.js@1.1.0/js/impress.js
+- Source: https://cdn.jsdelivr.net/gh/impress/impress.js/js/impress.js
+
+### **Stable release:**
+Head to the releases tab and download the source code as zip or as a tarball. Go ahead and unzip / untar it. You may find the framework under ```/js/impress.js```. Do also include the ```/css/impress-common.css```, if you want to make your life a bit easier.
+
+
+## Setting up the project
+Open up your favorite text-editor / IDE, for example Visual Studio Code, Atom, Notepad ++, ...
+Now, copy the folder */js/* and the folder */css* (if you want to make your life a bit easier) into the folder you are working in and create a new file called *index.html* and create the basic HTML layout:
+
+```
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>My first presentation</title>
+        <link rel="stylesheet" href="/css/impress-common.css"><!--Leave out, if you don't use impress-common.css-->
+    </head>
+    <body class="impress-not-supported">
+        <div class="fallback-message">
+            <p>Your browser <b>doesn't support the features required</b> by impress.js, so you are presented with a simplified version of this presentation.</p>
+            <p>For the best experience please use the latest <b>Chrome</b>, <b>Safari</b> or <b>Firefox</b> browser.</p>
+        </div>
+
+        <div id="impress">
+            <div id="myFirstSlide" class="step">
+                <h1>My first Slide</h1>
+            </div>
+        </div>
+
+        <script src="https://cdn.jsdelivr.net/gh/impress/impress.js@2.0.0/js/impress.js"></script><!--Only use, if you have internet connection when giving the presentation!-->
+
+        <script type="text/javascript" src='./js/impress.js></script><!--Only use, if you are not using the cdn!-->
```

Comment:
Okay, last question...

Have you tested what happens if someone copies this as is, and keeps both of the script tags above? (And both files exist,)

Reply:
I have not, but I will do. I am pretty sure I already did something like that by accident with I believe jQuery but not with impress. It still worked

Reply:
I've checked on my iPad and it is working. I will check with LibreWolf as well (which is just Firefox with more privacy)

Reply:
I have a suggestion to root out any possibility of something bad happening. I will just comment one of them out and tell the user to just uncomment it.

Reply:
Okay, it caused problems, it skips steps (so only every other step is being displayed)

Reply:
Interesting. That's because it would load both of them, so 2 copies of impress both move forward 1 step, at the same moment.

Thanks for testing.

My suggestion would be to go back to this:

    <script> src="js/impress.js"></script>
    <script>window.impress || document.write('<script src="https://cdn.jsdelivr.net/gh/impress/impress.js@2.0.0/js/impress.js">\x3C/script>');</script>

Although it's a bit unreadable to a beginner, it has the benefit that it just works even if you don't understand it.

(I modified the above compared to previous discussion so that it first tries the local copy, and the remote is the fallback.)

Reply:
Yeah. It is. I assumed browser cache would get it, but this isn't the case. I will change it to what you suggested

Reply:
Ohh I just noticed a problem with the CDN... You need to have a server like apache, http-server (which is a node module), as network requests are not allowed in the file protocol afaik.

Reply:
oh nvm